src/odh-uploader/handler.ts

```diff
@@ -64,7 +64,8 @@ export const setMailOptions = (
         html: emailTemplate(
             uuid,
             passengerType,
-            new Date(Date.now()).toLocaleString(),
+            `${new Date().getDate()}-${new Date().getMonth() +
+                1}-${new Date().getFullYear()} ${new Date().toLocaleTimeString('en-GB')}`,
```

Comment:
`toLocaleTimeString` won't guarantee the time zone I don't think, it just puts it in a British format, we could use the moment timezone library as a quick way to handle this